docs/Remix.md

```diff
@@ -271,14 +269,57 @@ Note that the Supabase credentials never leave the server. It's up to you to add
 
 ## Remix v2
 
-This section is for projects using [the new routing file conventions (v2)](https://remix.run/docs/en/main/file-conventions/route-files-v2).
+### Setting Up Remix
+
+Let's start by creating a new Remix project. Run the following command:
+
+```sh
+npx create-remix@latest
+```
+
+This script will ask you for more details about your project. You can use the following options:
+
+- The name you want to give to your project, e.g. `remix-admin`
+- Initialize a new git repository? Choose Yes
+- Install dependencies with npm? Choose Yes
+
+The project structure should look like this:  
+
+![Remix v2 project structure](./img/remix-v2-structure.png)
+
+### Setting Up React-Admin In Remix
+
+Add the `react-admin` npm package, as well as a data provider package. In this example, we'll use `ra-data-json-server` to connect to a test API provided by [JSONPlaceholder](https://jsonplaceholder.typicode.com).
+
+```sh
+cd remix-admin
+npm add react-admin ra-data-json-server
+```
+**Tip**: If you're using yarn, Remix and react-admin both install `react-router`, and due to the way each library handles its dependencies, this results in duplicate packages. To avoid this, use [yarn resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) to force Remix to use the same version of `react-router` as react-admin. So add the following to the `package.json` file:
+
+```js
+{
+  // ...
+  "resolutions": {
+    "react-router": "6.8.1",
+    "react-router-dom": "6.8.1"
+  }
+}
+```
+
+And call `yarn` again to install the dependencies:
+
+```sh
+yarn
+```
 
 ### Using React-Admin As The Root Application
 
-If you want to serve the admin app component in the root path ('/'), edit the file called `_routes/index.tsx`, and replace the content with the following:
+Remix and react-admin both use [react-router](https://reactrouter.com/) for routing. React-admin detects when it is included inside an existing React Router context. If so, react-admin will reuses it. This is problematic because Remix uses file-based routing. So when react-admin changes the route to `/posts` for instance, Remix will look for a corresponding `app/routes/posts.tsx` file. As it doesn't exist, Remix will render a 404.
```

Comment:
```suggestion
Remix and react-admin both use [react-router](https://reactrouter.com/) for routing. React-admin detects when it is included inside an existing React Router context. If so, react-admin will reuse it. This is problematic because Remix uses file-based routing. So when react-admin changes the route to `/posts` for instance, Remix will look for a corresponding `app/routes/posts.tsx` file. As it doesn't exist, Remix will render a 404.
```